backend/src/schema/resolvers/notifications.js

```diff
@@ -41,13 +40,15 @@ export default {
         default:
           orderByClause = ''
       }
-
+      const offset = args.offset && typeof args.offset === 'number' ? `SKIP ${args.offset}` : ''
+      const limit = args.first && typeof args.first === 'number' ? `LIMIT ${args.first}` : ''
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-29T22:09:41Z" title="Tuesday, October 29th 2019, 11:09:41 pm +01:00">Oct 29, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -41,13 +40,15 @@ export default {
         default:
           orderByClause = ''
       }
-
+      const offset = args.offset ? `SKIP ${args.offset}` : ''
+      const limit = args.first ? `LIMIT ${args.first}` : ''
```


Our GraphQL schema saves us here, because you must enter an Integer. But if that wouldn't be the case, we would have a cypher injection vulnerability here.